.github/workflows/ci.yml

```diff
@@ -366,7 +365,6 @@ jobs:
         with:
           persist-credentials: false
       - uses: Alfresco/alfresco-build-tools/.github/actions/get-build-info@v1.35.2
-      - uses: Alfresco/alfresco-build-tools/.github/actions/free-hosted-runner-disk-space@v1.35.2
```

Comment:
it is not worth anymore doing the cleanup? there is more space available with large runners?

Reply:
Yes, larger runners come with more disk space. What's more, the cleanup step on these runners might take couple of minutes:
https://github.com/Alfresco/acs-packaging/actions/runs/7084831853/job/19279893401#step:4